KristinJJ/GroupMovieRecommender
# Drag and drop functionality for Movie ranking

As a USER, I want to be able to rank movies by dragging and dropping, while seeing the movie posters so I know which movie I am ranking. API call data is now able to be sorted using Angular Material drag and drop, with custom preview of movie posters. 
closes #23 #15 

![2021-07-12 22 13 29](https://user-images.githubusercontent.com/26866132/125394827-22ca1780-e35f-11eb-90b7-c6a51a4d0d03.gif)

Tested on MacOS with Chrome browser

To test:
[ ] Drag the movie titles to see the drag preview of a movie poster 
[ ] Drop back among the movie titles to see the new ranking is saved
[ ] Change which movie is at the top to see the Top Choice change.
